client/state/reader/follows/reducer.js

```diff
@@ -158,6 +159,25 @@ export const items = createReducer(
 			);
 			return merge( {}, state, keyedNewFollows );
 		},
+		[ READER_SITE_REQUEST_SUCCESS ]: ( state, action ) => {
+			const incomingSite = action.payload;
+			if ( ! incomingSite || ! incomingSite.is_following ) {
+				return state;
+			}
+			const urlKey = prepareComparableUrl( incomingSite.feed_URL );
+			const currentFollow = state[ urlKey ];
+			const newFollow = {
```

Comment:
can you do the merge at currentFollow's initialization and skip it later?

Something like:

```js
const newFollow = {
   ...currentFollow,
   is_following: true,
   delivery_methods: get( incomingSite, '...'), 
}
return { ...state, [ urlKey ]: newFollow };
```

Reply:
`merge` does a deep merge, where spread is a shallow copy. Unsure if it matters here...

Reply:
Deep merge is used throughout this reducer, so best to leave it using `merge` for consistency's sake I think.